pom.xml

```
@@ -349,6 +349,18 @@
       </plugin>
       <!-- End publish to maven central -->
 
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.3.0</version>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>dev.openfeature.sdk</Automatic-Module-Name>
```

Minor remark - shall we move this name to pom's properties section?  

```suggestion
              <Automatic-Module-Name>${module-name}</Automatic-Module-Name>
```

where `module-name` is in properties.

---

Ya, this is what I was describing [here](https://github.com/open-feature/java-sdk-contrib/issues/261). I think you're right that it makes sense to do it here as well.

---

@toddbaert  @ARobertsCollibra  I added this suggestion :)